docs/how-monty-works/monty/reference-frame-transformations.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Reference Frame Transforms
+---
+In a sensorimotor learning setup one naturally encounters several different reference frames in which information can be represented. Additionally, Monty has it's internal reference frames in which is learns models of objects. Those can get confusing to wrap your head around and keep track of so here is a brief overview of all the reference frames involved in a typical Monty setup.
+
+# Reference Frames in a Typical Monty Experiment
+![](../../figures/how-monty-works/reference_frames_overview.png)
+
+### 🟢 Object rel. World
+The orientation of the object in the world (unknown to Monty). For instance, if we use a simulator, this would be the configuration that specifies where the object is placed in the environment (like we do [here](https://github.com/thousandbrainsproject/tbp.monty/blob/4844ef17a4cadce455acb8d852fe3ed7038a298f/src/tbp/monty/frameworks/config_utils/make_dataset_configs.py#L229)). In the real world we don't have a specific origin but still, all objects are placed in a common reference frame with relative displacements between each other.
+### 🟢⚫️ Feature rel. Object
+The pose of all the features on the object (also unknown to the Monty). For instance, if you use a 3D simulator, this could be defined in the object meshes.
```

Comment:
suggestion: withheld


```suggestion
The pose of all the features on the object (also withheld from Monty). For instance, if you use a 3D simulator, this could be defined in the object meshes.
```